src/nav/dashboards.yml

```diff
@@ -51,6 +51,12 @@ pages:
             path: /docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql 
           - title: NRQL tutorial - Your first queries
             path: /docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial
+          - title: NRQL tutorial - Control your data
```

Comment:
I don't think we use `-`s like this across the docs site. Can we switch to `:`, which is a bit more standard?